software/redlib.yml

```diff
@@ -0,0 +1,12 @@
+name: Redlib
+website_url: https://github.com/redlib-org/redlib
+description: An alternative private front-end to Reddit, with its origins in Libreddit.
+licenses:
+  - AGPL-3.0
+platforms:
+  - Rust
+tags:
+  - Communication - Social Networks and Forums
+source_code_url: https://github.com/redlib-org/redlib
+depends_3rdparty: true
+archived: false
```

Comment:
Optional so can be removed.
```suggestion
```

Reply:
`depends_3rdparty: true` is only optional, if it is `false`. Since in this case it is true, it is needed.

Reply:
ah yes, my mistake, I misread it!